src/Components/Results/Results.js

```diff
@@ -0,0 +1,65 @@
+import React, {useState, useEffect} from 'react'
+import Card from '../Card/Card'
+import './Results.css'
+import { searchAlbums } from '../../api'
+
+function Results({query}) {
+  const [results, setResults] = useState([])
+  const [pagination, setPagination] = useState({page: 1, pages: 1})
+  const [isLoading, setLoading] = useState(true)
+
+  useEffect(() => {
+    setLoading(true)
+    searchAlbums(query, 1)
+      .then(data => {
+        setResults(data.results)
+        setPagination(data.pagination)
+        setLoading(false)
+      })
+      .catch(error => alert(error.message))
```

Comment:
Displaying this on the DOM instead of having an alert box pop up would be less annoying for the user. We could easily have an error state and have a conditional if true, display the `error.message`

Reply:
Yeah, this was a sort of makeshift solution until the other components are in place/ we decide where that error state should go.